docs/en/02_Developer_Guides/19_GraphQL/01_getting_started/05_deploying_the_schema.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Deploying the schema
+summary: Deploy your GraphQL schema to a test or production environment
+icon: rocket
+---
+
+# Getting started
+
+[CHILDREN asList]
+
+[alert]
+You are viewing docs for a pre-release version of silverstripe/graphql (4.x).
```

Comment:
I'd drop the pre-release bit, it's very close to being stable

Reply:
Removing that from all the docs globally is a separate issue. I think it pays to be consistent and have the warning on _all_ docs until we remove it from _all_ docs.